.gitlab-ci.yml

```diff
@@ -223,6 +223,9 @@ build-nightly:
     GIT_STRATEGY:                  none
     DOCKERFILE:                    ci.Dockerfile
     IMAGE_NAME:                    docker.io/paritytech/$CI_JOB_NAME
+    VAULT_SERVER_URL: "https://vault.parity-mgmt-vault.parity.io"
+    VAULT_AUTH_PATH: "gitlab-parity-io-jwt"
+    VAULT_AUTH_ROLE: "cicd_gitlab_parity_${CI_PROJECT_NAME}"
```

Comment:
Please either fix alingment in the added lines or get rid of the alignment in others to keep it consistent.

Reply:
Formatting fixed.

What goes to GitLab settings vs Vault:
For example secrets stored in GitLab are non-auditable and non-versionable.